avatar/avatar.css

```diff
@@ -0,0 +1,95 @@
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+.avatar-container{
+    flex-grow: 3;
+    flex-shrink: 1;
+}
+.avatar-examples{
+    align-items: center;
+    padding: 2rem;
+    gap: 2rem;
+    flex-wrap: wrap;
+    margin: 1rem;
+}
+.avatar{
+    border-radius: 50%;
+    border: 5px solid var(--primary-color);
+}
+.avatar-lg{
+    width: 12rem;
+    height: 12rem;
+}
+.avatar-md{
+    width: 9rem;
+    height: 9rem;
+}
+.avatar-sm{
+    width: 6rem;
+    height: 6rem;
+}
+.avatar-xs{
+    width: 3rem;
+    height: 3rem;
+}
+.avatar-text{
```

Comment:
The size of text is same for all avatars, you should have created variations in different avatars